src/vue/forms/InvestForm.vue

```diff
@@ -158,23 +158,6 @@
       </form>
     </template>
 
-    <template v-else-if="isLoaded && isAllowedAccountType">
-      <no-data-message
-        icon-name="alert-circle"
-        :title="'invest-form.insufficient-balance-title' | globalize"
-        :message="'invest-form.insufficient-balance-desc' | globalize"
```

Comment:
You did not remove this `invest-form.insufficient-balance-title` `invest-form.insufficient-balance-desc` translations.

Reply:
It's not in `reit-en.json`